src/Symfony/Component/Serializer/Encoder/JsonDecode.php

```diff
@@ -57,14 +60,15 @@ public function __construct($defaultContext = [], int $depth = 512)
         }
 
         $this->defaultContext = array_merge($this->defaultContext, $defaultContext);
+        $this->propertyAccessor = PropertyAccess::createPropertyAccessor();
```

Comment:
It adds a hard dependency on property access which is still a suggested dependency in the composer